pxtsim/simdriver.ts

```diff
@@ -333,6 +333,8 @@ namespace pxsim {
                             this.container.appendChild(wrapper);
                             messageFrame = wrapper.firstElementChild as HTMLIFrameElement;
                             messageFrame.dataset[FRAME_DATA_MESSAGE_CHANNEL] = messageChannel;
+                            pxsim.U.addClass(wrapper, "simmsg")
+                            pxsim.U.addClass(wrapper, "simmsg" + messageChannel)
```

Comment:
wouldn't it be better to have "simmsg " + messageChannel so you get two classes? or you just don't want the sim styling to apply to jd iframe?

Reply:
in less, in have ``.simmsg > iframe { ... }`` to skin the any message simulator iframe and ``.simmsgjacdac > iframe { ... }`` for jacdac specifics.